### DISCUSSION TITLE
New Groups cannot view existing recipes - Creating a View-Only Group

### DISCUSSION THREAD

**Discussion by ldrach** [mealie-recipes/mealie#3567] [Q&A] [answered]
Hello,

I have been using mealie a lot lately and I want to add some of my family members to the app. Ideally I would like them to only have view access, so I created another group for view-only. In theory, this seems like it should be pretty simple, but I'm wondering if I am missing something because any time I try to set this group up, they are not able to see the existing recipes I have already created using the admin group.

Mealie Version: 1.5.1
Running in a Docker container

These are the settings I am using for the admin group

Admin:
**Private Group**: Checked (I have tried checked and unchecked)
**Allow users outside of your group you see your recipes**: Checked
**Show nutritional information**: checked
**Show recipe assets**: checked
**Default to landscape view**: unchecked
**Disable users from commenting on recipes**: unchecked
**Disable organizing recipe ingredients by units and food**: unchecked

These are the settings I am using for the View only role:

View-Only:
**Private Group**: unchecked (I have tried checked and unchecked)
**Allow users outside of your group you see your recipes**: Checked  (I have tried checked and unchecked)
**Show nutritional information**: checked
**Show recipe assets**: checked
**Default to landscape view**: unchecked
**Disable users from commenting on recipes**: unchecked
**Disable organizing recipe ingredients by units and food**: unchecked  (I have tried checked and unchecked)

I have also tested this with both an admin and non-admin user with the same results. For some reason, no matter what I do, the View-Only role is never able to see any recipes. If anyone has some insight on the matter, I would appreciate the help! I feel like I have messed around with every possible setting at this point and had no luck!

Thanks!

**Comment by michael-genson** [comment, marked as answer]
Groups are fully isolated instances; there is no sharing between groups. Currently there is no way to create users that can't create recipes, however you can make your entire instance public which will allow anyone to view recipes without an account (which seems to fit your use case, as long as you don't mind anyone with a link being able to see recipes):

https://docs.mealie.io/documentation/getting-started/usage/permissions-and-public-access/#public-recipe-access

**Reply by ldrach** [reply]
Hey, thank you for the information! I will probably go ahead and just make the recipes public if that is the case.

Out of curiosity, what exactly is the function of the setting "Allow users outside of your group you see your recipes" then? I think that must have been what confused me!

**Reply by michael-genson** [reply]
The reason there are two settings is because the "make group private" is sort of a global toggle; if your group is private, all other privacy settings are overridden. The other settings are more fine-tuned (e.g. recipe access)

**Comment by bf8392** [comment]
A view only group would be a great feature! Is it planned? I want to share receipes with another houshold, but don't want them to be able to edit categories or food, and some of them not even the receipes...how can I implement without making the whole instance public?

**Reply by michael-genson** [reply]
At this time anything cross-group isn't planned. You can lock recipes from being edited by non-household members (there is a toggle in household settings):
![image](https://github.com/user-attachments/assets/a9832340-838e-4a81-89f6-f09f46ea719d)

You can also prevent users from editing organizers (categories, tags, foods, etc.) with user settings:
![image](https://github.com/user-attachments/assets/63056027-2ceb-4f97-a955-399f29d04916)

Note that this is global for that user, so they can't organize _any_ recipes. At this time there's no way to limit organization to one's own household. I could see this being implemented in the future, though

**Reply by bf8392** [reply]
How did you get to this view? That's exactly what I'm looking for; 
![459636439-63056027-2ceb-4f97-a955-399f29d04916](https://github.com/user-attachments/assets/75ba44af-f9bd-4846-9fcc-25d44162edc9)

**Reply by bf8392** [reply]
I found it and tried with a user ("Benutzer") to deactivate it, but he can still make categories and foods 
![Screenshot 2025-06-27 012118](https://github.com/user-attachments/assets/0d70630d-e02f-4323-895c-89dd4c7870d5)
